### PR TITLE
Move build_tar to pkg/private/tar

### PR DESCRIPTION
**Pull request by aiuto**
Just one more cleanup CL towards silo-ing each package type

### CODE REVIEW

**Comment by nacl** [comment]
Looks like you're missing `pkg/private/tar/BUILD`

**Comment by aiuto** [comment]
I added the missing file.

File: pkg/private/tar/BUILD [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
```
Comment (nacl) [inline pkg/private/tar/BUILD:1]: Does this need a new copyright year?

File: pkg/private/tar/BUILD [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        "*.bzl",
+        "*.py",
+    ]),
+    visibility = ["//distro:__pkg__"],
+)
+
+exports_files(
+    glob([
+        "*.bzl",
+    ]),
```
Comment (nacl) [inline pkg/private/tar/BUILD:37]: Nit: can just be one line

File: pkg/private/tar/BUILD [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    srcs = [
+        "BUILD",
+    ] + glob([
+        "*.bzl",
+        "*.py",
+    ]),
+    visibility = ["//distro:__pkg__"],
+)
+
+exports_files(
```
Comment (nacl) [inline pkg/private/tar/BUILD:34]: A style I've found to work well is to have things like `exports_files` be at the top of the file, followed by any other rules.  Makes the special nature of them a little more obvious.

Nothing wrong with this, though.

**Review by nacl — APPROVED** [review]
LGTM.  See style nits below.

File: pkg/private/tar/BUILD (hunk 2 above)
Comment (aiuto) [reply pkg/private/tar/BUILD:37]: done

File: pkg/private/tar/BUILD (hunk 3 above)
Comment (aiuto) [reply pkg/private/tar/BUILD:34]: The fact that these have to be exported at all, just to please bzl_library is a failure.

**Pull request merged**